src/test/java/com/hocs/test/glue/PrivateOfficeSignOffStepDefs.java

```diff
@@ -33,13 +33,22 @@
 
     @When("^I complete the Private Office stage$")
     public void completePrivateOfficeStage() {
-        dataInput.selectTeam1();
-        successfulCaseCreation.clickSessionVariableViaLinkText();
-        System.out.println("Selecting Accept");
+        homepage.findMyPrivateOfficeCase();
+        homepage.selectAllocationUserByVisibleText("Danny Large (danny.large@ten10.com)");
+        homepage.selectMyCases();
+        successfulCaseCreation.selectCaseReferenceNumberViaLinkText();
         privateOffice.clickPrivateOfficeAcceptRadioButton();
-        System.out.println("Private office accepts this draft, returning to home page.");
         page.clickFinishButton();
+    }
 
+    @When("^The case is rejected at the Private Office stage$")
+    public void rejectAtPrivateOffice() {
+        homepage.findMyPrivateOfficeCase();
+        homepage.selectAllocationUserByVisibleText("Danny Large (danny.large@ten10.com)");
+        homepage.selectMyCases();
+        successfulCaseCreation.selectCaseReferenceNumberViaLinkText();
+        privateOffice.clickPrivateOfficeRejectRadioButton();
+        page.clickFinishButton();
```

Comment:
One of the things the guys have been saying is if you are using the page class you should direct to the page you are on, as they all extend page.  so here would be `privateOffice.clickFinishButton();`